giftless/storage/google_cloud.py

```diff
@@ -1,10 +1,15 @@
+import binascii
+import collections
+import hashlib
 import json
+import logging as log
```

Comment:
I would refrain from using an alias for something so common as `logging` unless there is a very good reason such as a naming conflict. This makes the code much less readable.

Reply:
removed